plugins/tags-resources/src/components/CreateTagElement.svelte

```diff
@@ -94,13 +94,20 @@
       }
     )
   }
+  
+
+  function isNotEmpty(str: string): boolean {
```

Comment:
It is idiomatic for use to write something like
```
$: canSave = title.trim().length > 0
```
I'm also not sure regarding `undefined` check, the `str` and `title` types do not assume `undefined` value, so either remove it or fix types.

Reply:
@aonnikov 
Thank you for the clarification. Given that the variable title is explicitly declared as a string, an undefined check may indeed be unnecessary for string validation in this specific context. Modifying the PR accordingly.